ephios/core/templates/core/fragments/signup_method_missing.html

```diff
@@ -0,0 +1,11 @@
+{% load i18n %}
+
+<p class="text-muted">
+    {% translate "We're sorry! There is a problem displaying the state of the shift." %}
+</p>
+{% if request.user.is_superuser %}
+    <p class="text-muted">
+        Here's what we know:
+        {{ exception_message }}
```

Comment:
this would be a first. currently we don't display any exceptions inside the application, even for superusers. I think it is best practice to keep it that way

Reply:
Oh yeah I wanted to remove that anyway. For the reasons you state.